include/merlin_hashtable.cuh

```diff
@@ -239,7 +239,7 @@ class HashTable {
    * the evict strategy. If false, it requires the metas follow the evict
    * strategy of table.
    */
-  void insert_or_assign(size_type n,
+  void insert_or_assign(size_t n,
```

Comment:
Pls keep `size_type` here, follow the style of std::unordered_map

Reply:
The usage of s`ize_type` is not consequent. This value is converted to size_t in the function.

Reply:
You realize that the entire TableCore and all kernels don't have the size argument templated. Right?

Reply:
Ok, Agreed!